frontend/web/styles/3rdParty/_hljs.scss

```
@@ -138,11 +143,11 @@
   user-select: none;
   font-family: Menlo, Monaco, Consolas, 'Liberation Mono', 'Courier New',
     monospace;
-  background-color: $code-btn-bg;
+  background-color: var(--color-surface-muted);
   font-family: $font-family-header;
   overflow: hidden;
   @include transition(all 0.2s ease-in-out);
-  color: $btn-code-button-color;
+  color: var(--color-text-default);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)